apps/src/templates/rubrics/RubricContent.jsx

```diff
@@ -205,17 +206,21 @@ export default function RubricContent({
         )}
       </div>
       {experiments.isEnabled('ai-rubrics-redesign') ? (
-        <LearningGoals
-          learningGoals={rubric.learningGoals}
-          teacherHasEnabledAi={teacherHasEnabledAi}
-          canProvideFeedback={canProvideFeedback}
-          reportingData={reportingData}
-          studentLevelInfo={studentLevelInfo}
-          isStudent={false}
-          feedbackAdded={feedbackAdded}
-          setFeedbackAdded={setFeedbackAdded}
-          aiEvaluations={aiEvaluations}
-        />
+        // TODO: remove tempContainer div when experiment is ready to roll out
```

Comment:
Nice - is this tracked anywhere?

Reply:
Not yet, but I have a couple of tickets to add, so this will be one.